From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: chickeneer <emcchickeneer@gmail.com>
Date: Wed, 8 Jun 2016 20:39:19 -0400
Subject: [PATCH] Fix Potion Equals Bug

Will be in Spigot 1.10, but getting it in for 1.9 users

diff --git a/src/main/java/org/bukkit/craftbukkit/potion/CraftPotionUtil.java b/src/main/java/org/bukkit/craftbukkit/potion/CraftPotionUtil.java
index 0000000000000000000000000000000000000000..0000000000000000000000000000000000000000 100644
--- a/src/main/java/org/bukkit/craftbukkit/potion/CraftPotionUtil.java
+++ b/src/main/java/org/bukkit/craftbukkit/potion/CraftPotionUtil.java
@@ -0,0 +0,0 @@ public class CraftPotionUtil {
             .build();
 
     public static String fromBukkit(PotionData data) {
+        String type;
         if (data.isUpgraded()) {
-            return upgradeable.get(data.getType());
+            type = upgradeable.get(data.getType());
+        } else if (data.isExtended()) {
+            type = extendable.get(data.getType());
+        } else {
+            type = regular.get(data.getType());
         }
-        if (data.isExtended()) {
-            return extendable.get(data.getType());
-        }
-        return regular.get(data.getType());
+        return "minecraft:" + type;
     }
 
     public static PotionData toBukkit(String type) {
--